src/index.js

```diff
@@ -0,0 +1,163 @@
+import './styles/styles.css';
+import TaskObject from '../modules/taskObj.js';
+// eslint-disable-next-line import/no-cycle
```

Comment:
- [ ] [OPTIONAL] Kindly consider fixing linters instead of ignoring the errors, linters can make your code clean